examples/bri-3/src/bri/transactions/agents/mockTransactionStorage.agent.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
```

Comment:
since this is a mock agent, can we segregate this from main code and move inside a spec folder?

Reply:
You usually want to keep the mocks next to the things they mock since that is the scope they are interested in. Similar logic to where we place the spec file.

Reply:
yeah, but that doesn't stop us from structuring these inside a spec folder close to the class

Reply:
storage mocks are a temporary solution as we learn prisma. They are cumbersome to maintain already and will become more so as we move forward. We ll probably have to ditch them and use a mocking framework instead. Also, we 've got spec files that do not have associated mock files, so having them in a separate folder sounds like an unnecessary step. I woould rather keep the state as it is atm, as we ll end up with a standardized approach of spec files next to the thing they test eventually. If you have a strong motivation for this change i can do it, but i do not see additional value in doing it.